entrypoint.sh

```diff
@@ -11,14 +11,11 @@ echo Running makemigrations
 echo Running migrations
 ./manage.py migrate
 
-echo "Ensuring our base configuration is present in the database"
+echo Ensuring our base configuration is present in the database
 ./manage.py ensure_initial_site_configuration
 
-echo "Initializing difficulty values"
-./manage.py initialize_difficulty_values
```

Comment:
I think this is reasonable but do we have an easy way to trigger this otherwise? I'm wondering whether that should be something like a nightly celery task or an admin button.

Reply:
I will create a different way to trigger this.